webapp/src/Controller/Jury/ExecutableController.php

```diff
@@ -268,7 +268,26 @@ public function deleteSingleAction(Request $request, string $execId, int $rank):
 
             return $this->render('jury/delete.html.twig', $data);
         } else {
-            $this->em->remove($file);
+            // Create a copy of all files except $file
```

Comment:
The `$file` in this comment is the `$file from above` and then in the loop below is a different `$file`. While this is valid code, it is confusing.

I suggest renaming `$file` to `$fileToDelete`, and `$rank` to `$rankToDelete`.